### PR TITLE
[Ja] translated rss

### PR DESCRIPTION
**Pull request by junichirojp**
I translated RSS.
Maybe this file is the last english file.

### CODE REVIEW

File: ja/views/helpers/rss.rst [hunk 1]
```diff
@@ -1,22 +1,181 @@
...
+いくつかの簡単なステップの後、 ``articles/index`` に拡張子 (.rss) を追加し、
+``articles/index.rss`` という URL を作成します。
+このウェブサービスを公開して実行する前にいくつか行う必要があります。
+最初に拡張子パースを有効にします。これは **config/routes.php** で行います。
+::
+
+    Router::extensions('rss');
+
+上記のように宣言すると、.rss 拡張子を利用できます。
+:php:meth:`Cake\\Routing\\Router::extensions()` を使用すると、
```
Comment (okinaka) [inline ja/views/helpers/rss.rst]: > 使用すると、

使用する時、

**Comment by chinpei215** [comment]
@junichirojp When someone is reviewing your pull request, basically, you shouldn't merge it by yourself. Since it is just a document, you might have thought optimistic about it though. I appreciate if you review someone's pull requests, and merge his/her pull request.

File: ja/views/helpers/rss.rst (hunk 1 above)
Comment (junichirojp) [inline ja/views/helpers/rss.rst]: 👍

**Comment by junichirojp** [comment]
@chinpei215 thanks reply.
Of cause i agree with you and have same mind.
And as you say, i thing frankly because the review is just one word modified.

使用すると→使用する時

Don't worry about my contribution mind. i usually never merge my pull request.

**Comment by chinpei215** [comment]
You have overlooked at the commit history. From the next time, please squash your commits before sending a pull request, if there are no important changes that should be noted.

**Pull request merged**